pyCGM_Single/pyCGM.py

```diff
@@ -1812,20 +1834,20 @@ def wristJointCenter(frame,shoulderJC,wand,elbowJC):
     return [origin,axis]
 
 def handJointCenter(frame,elbowJC,wristJC,vsk=None):
-    """Calculate the Hand joint axis ( Hand) function.
+    """Calculate the Hand joint axis (Hand).
 
-    Takes in a dictionary of x,y,z positions and marker names.
-    and takes the elbow axis and wrist axis.
+    Takes in a dictionary of marker names to x, y, z positions, wrist axis
+    subject measurements from a VSK file.
```

Comment:
doesn't need to be from a 'vsk file', its really a dictionary of subject measurements.